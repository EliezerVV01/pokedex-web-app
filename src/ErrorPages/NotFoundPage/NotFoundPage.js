import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import Button from './../../components/Button/Button';

import Form from './../../components/Form/Form'
import './NotFoundPage.css';
import PokemonSad from '../../Assets/Images/pokemon_sad.png';

class NotFoundPage extends Component {
    render() {
        return (
            <div className="NotFoundPage">
                <Form margin_top="100px">
                    <img alt="pokemon_sad"
                        src={PokemonSad}
                        className="pokemon_sad" />
                    <h3>Error 404: Not Found Page</h3>
                    <NavLink to="/" exact><Button className="btn btn-primary">Go to Home Page!</Button> </NavLink>
                </Form>
            </div>
        )
    }
}

export default NotFoundPage;