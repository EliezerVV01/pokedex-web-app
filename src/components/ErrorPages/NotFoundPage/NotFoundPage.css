.NotFoundPage {
    position: absolute;
    height: 100%; 
    width: 100%;
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    background-image: url("/static/media/pokemon_wallpaper.22a66419.jpg");
    text-align: center;
}

@media (max-width: 500px){
    .NotFoundPage{
        background-image: none;
          background: white;
        }
}

.pokemon_sad{
    width: 100%;
    margin-bottom: 25px;
}

