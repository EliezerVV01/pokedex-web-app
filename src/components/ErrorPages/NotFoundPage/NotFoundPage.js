import React, { Component } from 'react';
import Button from './../../Button/Button';
import Form from './../../Form/Form'
import './NotFoundPage.css';
import PokemonSad from '../../../Assets/Images/pokemon_sad.png';
import * as links from './../../../Routes/RoutesList';

class NotFoundPage extends Component {
    render() {
        return (
            <div className="NotFoundPage">
                <Form margin_top="100px">
                    <img alt="pokemon_sad"
                        src={PokemonSad}
                        className="pokemon_sad" />
                    <h3>Error 404: Not Found Page</h3>
                   <Button className="btn btn-primary" clicked={()=>this.props.history.push({pathname: links.HOME})}>Go to Home Page!</Button>
                </Form>
            </div>
        )
    }
}

export default NotFoundPage;