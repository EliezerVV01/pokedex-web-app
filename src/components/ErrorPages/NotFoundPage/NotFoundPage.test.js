import React from 'react';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<NotFoundPage></NotFoundPage>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});