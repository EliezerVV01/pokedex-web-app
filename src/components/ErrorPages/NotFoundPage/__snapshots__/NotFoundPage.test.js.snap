// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="NotFoundPage"
>
  <form
    style={
      Object {
        "marginTop": "100px",
      }
    }
  >
    <img
      alt="pokemon_sad"
      className="pokemon_sad"
      src="pokemon_sad.png"
    />
    <h3>
      Error 404: Not Found Page
    </h3>
    <button
      className="btn btn-primary Button"
      onClick={[Function]}
    >
      Go to Home Page!
    </button>
  </form>
</div>
`;
